duckduckgo/Android
# Feature/more thorough webview data clean

Task/Issue URL: https://app.asana.com/0/608920331025315/942179575131652
Tech Design URL: 
CC: 

**Description**:
Deletes the `WebView` data dir within our app's data dir when clearing all data. 

Also, measures some of the key areas when data clearing so we can see a breakdown of what is being done and how long it is taking.

⚠️ This needs tested for a little while before release, so we can see how it performs in `develop` after some use, before it is part of a release.

**Steps to test this PR**:
1. Use an emulator (or rooted device)
1. Browse the web for a bit
1. Visit http://dev-test.nemikor.com/web-storage/support-test/ and run web storage tests (will take a minute)
1. Verify `data/data/com.duckduckgo.mobile.android/app_webview` directory has lots in it
1. Perform a clear using the 🔥 button
1. Verify `data/data/com.duckduckgo.mobile.android/app_webview` directory is initialized back to a clean state
1. Verify the browser still works!


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
